JSDemos/Demos/Toolbar/Multiline/React/App.js

```diff
@@ -0,0 +1,335 @@
+} from './data.js';
+
+function getTextAlignItems(isExtended) {
+  return !isExtended
+    ? textAligns.map(({ icon, alignment, hint }) => ({ icon, alignment, hint }))
+    : textAligns;
+}
+
+const toolbarLineModes = ['Multiline mode', 'Singleline mode'];
+
+const attachButtonOptions = {
+  icon: 'attach',
+  text: 'Attach',
+  onClick: () => {
+    notify('Attach button has been clicked!');
+  },
+};
+
+const addButtonOptions = {
+  icon: 'add',
+  text: 'Add',
+  onClick: () => {
+    notify('Add button has been clicked!');
+  },
+};
+
+const removeButtonOptions = {
+  icon: 'trash',
+  text: 'Remove',
+  onClick: () => {
+    notify('Remove button has been clicked!');
+  },
+};
+
+const aboutButtonOptions = {
+  icon: 'help',
+  text: 'About',
+  onClick: () => {
+    notify('About button has been clicked!');
+  },
+};
+
+function App() {
+  const [lineHeight, setLineHeight] = React.useState(lineHeights[1].lineHeight);
+  const [fontSize, setFontSize] = React.useState(fontSizes[2].size);
+  const [textAlign, setTextAlign] = React.useState([textAligns[0].alignment]);
+  const [toolbarLineMode, setToolbarLineMode] = React.useState(
+    toolbarLineModes[0],
+  );
+  const [multiline, setMultiline] = React.useState(true);
+  const [checkBoxValue, setCheckBoxValue] = React.useState(false);
+
+  const onCheckBoxChange = React.useCallback(({ value }) => {
+    setCheckBoxValue(value);
+    notify('Checkbox option has been changed!');
+  }, []);
+
+  const onLineHeightChange = React.useCallback(
+    (e) => setLineHeight(e.item.lineHeight),
+    [],
+  );
+
+  const onFontSizeChange = React.useCallback(
+    (e) => setFontSize(e.item.size),
+    [],
+  );
+
+  const onTextAlignChange = React.useCallback((e) => {
+    setTextAlign([e.itemData.alignment]);
+  }, []);
+
+  const onToolbarLineModeChange = React.useCallback(({ value }) => {
+    setMultiline(value === toolbarLineModes[0]);
+    setToolbarLineMode(value);
+  }, []);
+
+  const renderFontSize = React.useCallback(
+    (itemData) => (
+      <div style={{ fontSize: `${itemData.size}px` }}>{itemData.text}</div>
+    ),
+    [],
+  );
+
+  const renderTextAlign = React.useCallback(() => {
+    const isMenu = false;
+
+    return (
+      <ButtonGroup
+        items={getTextAlignItems(isMenu)}
+        keyExpr="alignment"
+        stylingMode="outlined"
+        selectedItemKeys={textAlign}
+        onItemClick={onTextAlignChange}
+      ></ButtonGroup>
+    );
+  }, [textAlign]);
+
+  const renderTextAlignMenu = React.useCallback(() => {
+    const isMenu = true;
+
+    return (
+      <ButtonGroup
+        items={getTextAlignItems(isMenu)}
+        displayExpr="text"
+        keyExpr="alignment"
+        stylingMode="outlined"
+        selectedItemKeys={textAlign}
+        onItemClick={onTextAlignChange}
+      ></ButtonGroup>
+    );
+  }, [textAlign]);
+
+  const renderMenuSeparator = React.useCallback(() => <div className="menu-separator"></div>, []);
+
+  return (
+    <React.Fragment>
```

Comment:
Is it require to wrap resizable in react.fragment here?

Reply:
Yes, we need it